test/new-e2e/tests/installer/package_agent_test.go

```diff
@@ -101,8 +100,6 @@ func (s *packageAgentSuite) TestUpgrade_AgentDebRPM_to_OCI() {
 
 // TestUpgrade_Agent_OCI_then_DebRpm agent deb/rpm install while OCI one is installed
 func (s *packageAgentSuite) TestUpgrade_Agent_OCI_then_DebRpm() {
-	// incident-28477
-	flake.Mark(s.T())
```

Comment:
Same question here but I suppose it's part of the same change

Reply:
This test is fixed by the bump. The previous pipeline ran successfully without the marker, as the marker was on main but not on my branch